src/main/java/Bob/task/Task.java

```diff
@@ -0,0 +1,49 @@
+package bob.task;
+
+public class Task {
```

Comment:
Please include JavaDoc comments on the purpose of the Task class as well as in the subclasses inherited from this class (Todo, Event and Deadline).